bootloader/src/pyi_utils.c

```diff
@@ -850,21 +855,44 @@ set_systemd_env()
 pid_t child_pid = 0;
 
 static void
-_signal_handler(int signal)
+_ignore_signal(int signum)
+{
+    VS("LOADER: Ignoring signal %d\n", signum);
+}
+
+static void
+_signal_handler(int signum)
 {
-    kill(child_pid, signal);
+    VS("LOADER: Forwarding signal %d to child pid %d\n", signum, child_pid);
+    kill(child_pid, signum);
```

Comment:
Renaming parameters is a non-functional change, please revert this. This will increase the patch-sice and the time to review and understand what this change is doing.

Reply:
The previous name, `signal`, was shadowing a function name. Like you, I'm interested in the long-term maintainability of the code I write, so I fixed this issue. I've already split this tiny function into two handlers; even calling this a rename is a stretch.